htdocs/manage/tracking/user.bml

```diff
@@ -72,8 +68,9 @@ body<=
             event   => "JournalNewComment",
             journal => $journal,
             flags   => LJ::Subscription::TRACKING,
-        ) if $remote->can_track_all_community_comments( $journal );
-    
+        ) if $remote->can_track_all_community_comments( $journal )
+            and $journal->statusvis eq "V";
```

Comment:
Prefer to use `&&` unless you need the lower precedence operator for some reason.